core/templates/services/context.service.spec.ts

```diff
@@ -266,5 +375,19 @@ describe('Context service', () => {
       expect(ecs.getPageContext()).toBe('other');
     }
     );
+
+    it('should retrive exploration preview mode', () => {
```

Comment:
Type: retrieve

Reply:
What about 'should detect the editor tab context is preview'?

Reply:
Done!